bindings/pyroot/pythonizations/test/helper.py

```diff
@@ -0,0 +1,219 @@
+    @ROOT.Numba.Declare(['int', 'bool'], 'long')
+    def random_long(bits, signed):
+        if signed:
+            low = -1*2**(bits - 1)
+            high = 2**(bits - 1) -1
+        else:
+            low = 0
+            high = 2**bits
+        return np.random.randint(low, high)
+
+    def generate_integers(N, bits, signed):
+        if signed:
+            low = -2**(bits - 1)
+            high = 2**(bits - 1) -1
+        else:
+            low = 0
+            high = 2**bits
+
+        if not (bits == 64 and signed == False):
+            return np.random.randint(low, high, size  = N)
+        
+        # Numpy cannot generate unsigned 64 bit integers.
+        x = []
+        for i in range(N):
+            x.append(random.randint(0, 2**64))
+        return x
+    
+    N = 100 # df with 100 entries
+
+    df = ROOT.RDataFrame(N)
+
+    """
+    Cannot test String, Char_t and UChar_t right now as:
+    1. The generation of it (which I assumed would be equivalent to 8bit integers is false).
+    2. They cannot be converted into numpy arrays thus hindering my current "test conditions" which takes the mean of the entries.
+    3. Issue in file issues.py 
+    """
+    # col_name = "Char_t"
+    # df = df.Define(col_name, f"({col_name}) Numba::random_long(8, true)")#.Snapshot(col_name, "./Data/" + col_name + ".root") 
+    # col_name = "UChar_t"
+    # df = df.Define(col_name, f"({col_name}) Numba::random_long(8, false)")#.Snapshot(col_name, "./Data/" + col_name + ".root") 
+
+    col_name = "Short_t"
+    df = df.Define(col_name, f"({col_name}) Numba::random_long(16, true)")
+
+    col_name = "UShort_t"
+    df = df.Define(col_name, f"({col_name}) Numba::random_long(16, false)")
+
+    col_name = "Int_t"
+    df = df.Define(col_name, f"({col_name}) Numba::random_long(32, true)") 
+
+    col_name = "UInt_t"
+    df = df.Define(col_name, f"({col_name}) Numba::random_long(32, false)")
+
+    col_name = "Float_t"
+    df = df.Define(col_name, f"({col_name}) gRandom->Gaus()")
+
+    col_name = "Float16_t"
+    df = df.Define(col_name, f"({col_name}) gRandom->Gaus()")
+
+    col_name = "Double_t"
+    df = df.Define(col_name, f"({col_name}) gRandom->Gaus()")
+
+    col_name = "Double32_t"
+    df = df.Define(col_name, f"({col_name}) gRandom->Gaus()")
+
+    """
+    I needed an alternative way to generate Long64 types as:
+    1. Could not use gRandom as gRandom only returns long and not long long it gave me 0s and the range was invalid
+    2. Doubts as to the differences between types Long64_t and Long_t ...
+    WorkAround:
+    I put the numbers generated in an array and put filled the tree with that rather than use define.
+    """
+    # df = df.Define("Long64_t", "(Long64_t) Numba::random_long(64, true)")
+    # df = df.Define("ULong64_t", "(Long64_t) Numba::random_long(64, true)")
+    # df = df.Define("Long_t", "(Long_t) gRandom->Integer(pow(2,63)-1)")
+    # # df = df.Define("ULong_t", "(ULong_t) gRandom->Integer(pow(2,63))") 
```

Comment:
This type of comment is probably better for the docstring. The commented code should not be there.